core/templates/pages/release-coordinator-page/release-coordinator-page.component.html

```diff
@@ -20,7 +20,7 @@
       <h3>Flush the Memory Cache</h3>
       <div class="oppia-action-buttons-container">
         <button mat-raised-button color="primary" (click)="flushMemoryCache()">Flush Cache</button>
-        <button mat-raised-button color="primary" (click)="getMemoryCacheProfile()">Get Memory Cache Profile</button>
+        <button class="get-cache-profile-button" mat-raised-button color="primary" (click)="getMemoryCacheProfile()">Get Memory Cache Profile</button>
```

Comment:
What is this change doing? It looks like it'll affect the release coordinator page, so please show proof of correctness for any affected functionality

Reply:
wasn't intended updated.